examples/CMakeLists.txt

```diff
@@ -98,10 +98,9 @@ if(BUILD_PYTHON_INTERFACE)
         collisions
         collision-with-point-clouds
         append-urdf-model-with-another-model
-        simulation-contact-dynamics)
-      if(PYTHON_VERSION_MAJOR EQUAL 3)
-        list(APPEND ${PROJECT_NAME}_PYTHON_EXAMPLES static-contact-dynamics)
-      endif()
+        simulation-contact-dynamics
+        # talos-simulation
```

Comment:
If it's commented, remove it?
```suggestion
```

Reply:
I kept it for tracking missing examples not covered by the CI.